tests/test_config.py

```diff
@@ -241,6 +242,23 @@ def test_duplicate_coords():
         ProjectPoints.regions(regions, res_file, sam_files)
 
 
+def test_sam_configs():
```

Comment:
If you have time, maybe consider adding a negative tests of the bad sam_files input types? Like a list/tuple of sam filepaths and a single extracted SAM config dictionary (instead of an id-to-config mapper).

Reply:
will do, looks like there are some other tests failing... I'll dig into that.

Reply:
@grantbuster, miss any big ones to test for?
https://github.com/NREL/reV/blob/2de9e54ecd6b2d6196a8223647f37d5ec4ce2e40/tests/test_config.py#L262

Reply:
I'm not sure, can you annotate what youre testing for in these? I dont see why the 1st, 2nd, and 3rd tests should raise an error? And yes can you add the case where there is a list of sam_files input? I think that was one of our previous use cases that we deprecated right?

Reply:
Ah I see. @MRossol looks good!